### PR TITLE
feat: implement ElementHandle.uploadFile for WebDriver BiDi

### PR DESCRIPTION
**Pull request by jrandolf-google**

### CODE REVIEW

**Review by OrKoN — COMMENTED** [review]
Let's remove expectations for `[input.spec] input tests input should upload the file` tests

**Comment by OrKoN** [comment]
@jrandolf have you added WPT tests already?

**Review by OrKoN — COMMENTED** [review]
Let's also update https://pptr.dev/webdriver-bidi and move uploadFile from unsupported to supported

**Comment by Lightning00Blade** [comment]
@jrandolf making as `feat` as we officially have (some) BiDi support.

File: test/TestExpectations.json [hunk 1]
```diff
@@ -395,6 +395,12 @@
     "parameters": ["cdp", "firefox"],
     "expectations": ["SKIP"]
   },
+  {
+    "testIdPattern": "[input.spec] input tests input *",
```
Comment (OrKoN) [inline test/TestExpectations.json]: this pattern looks to be too broad?

**Comment by OrKoN** [comment]
> > Let's also update https://pptr.dev/webdriver-bidi and move uploadFile from unsupported to supported
> 
> I'll add this in a separate docs PR. We have more features that we support.

Please include it in this PR so that it is releasable.

File: test/TestExpectations.json (hunk 1 above)
Comment (jrandolf) [reply test/TestExpectations.json]: It actually only tests file upload lol

Comment (OrKoN) [reply test/TestExpectations.json]: let's rename `input` to `ElementHandle.uploadFile` in the test file?

**Review by OrKoN — APPROVED** [review]

**Pull request merged**